tools/slang-unit-test/unit-test-special-scalar-reflection.cpp

```
@@ -0,0 +1,145 @@
+    SLANG_CHECK(gDataType != nullptr);
+    SLANG_CHECK(gDataType->getKind() == slang::TypeReflection::Kind::Resource);
+
+    auto resultType = gDataType->getResourceResultType();
+    SLANG_CHECK(resultType != nullptr);
+    SLANG_CHECK(resultType->getKind() == slang::TypeReflection::Kind::Struct);
+
+    auto bfField = resultType->getFieldByIndex(0);
+    auto e4Field = resultType->getFieldByIndex(1);
+    auto e5Field = resultType->getFieldByIndex(2);
+    auto ipField = resultType->getFieldByIndex(3);
+    auto upField = resultType->getFieldByIndex(4);
```

Cppcheck (2.20.0)

[style] The function 'sprintf_s' is never used.

(unusedFunction)